spread.yaml

```diff
@@ -259,6 +259,24 @@ suites:
         restore-each: |
             $TESTSLIB/reset.sh --reuse-core
 
+    tests/regression/:
+        summary: Regression tests for snapd
+        systems: [-ubuntu-core-16-64-fixme]
+        prepare: |
+            . $TESTSLIB/prepare.sh
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+                prepare_all_snap
```

Comment:
This won't work on the boards, the systems are `ubuntu-core-16-64`, `ubuntu-core-16-arm-32` and `ubuntu-core-16-arm-64`, maybe better to use the `external` backend as a filter.